stable/stackdriver-exporter/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Stackdriver exporter for Prometheus
 name: stackdriver-exporter
-version: 0.0.6
+version: 0.0.7
```

lets bump this to 1.0.0 since this chart is in stable folder

```suggestion
version: 1.0.0
```